docs/native-tokens/minting.md

```diff
@@ -187,17 +187,16 @@ Policies are the defining factor under which tokens can be minted. Only those in
 We'll make a separate sub-directory in our work directory to keep everything policy-wise separated and more organized.
 For further reading, please check [the official docs](https://docs.cardano.org/native-tokens/getting-started/#tokenmintingpolicies) or the [github page about multi-signature scripts](https://github.com/input-output-hk/cardano-node/blob/c6b574229f76627a058a7e559599d2fc3f40575d/doc/reference/simple-scripts.md).
 
-"`bash
+```bash
 mkdir policy
 ```
 
-:::note

```

Comment:
This should not be changed, as it represents "Note" admonition as per [style guide](https://developers.cardano.org/docs/portal-style-guide#admonitions)